pkgs/development/python-modules/ssort/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  pathspec,
+  pytestCheckHook,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "ssort";
+  version = "0.13.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "bwhmather";
+    repo = "ssort";
+    rev = "refs/tags/${version}";
+    hash = "sha256-P/FUayCC7KfXjtzclTPLhLw5o0bV4L98tes69w+038o=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pathspec ];
+
+  pythonImportsCheck = [ "ssort" ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pyyaml
+  ];
+
+  preCheck = ''
+    substituteInPlace tests/test_executable.py \
+      --replace-fail '["ssort"]' "['${placeholder "out"}/bin/ssort']" \
+      --replace-fail '["ssort",' "['${placeholder "out"}/bin/ssort',"
+  '';
```

Comment:
Can we use `export PATH=$out/bin:$PATH` instead?

Reply:
but this is ok too.

Reply:
> but this is ok too.

We need **consensus** to move forward. Either you wait for natsukium to answer. Or you can try his solution.
Goal is mostly to reach consensus. And yes, there are many different ways to do things. And there isn't one right.
I can only tell you what I would do. I would try his solution. And see if it works fine. But then, you do you. You could wait him. I know this is something basic and simple. And not important, yes... I know. Your solution works too.
I'm just trying to offer a shortcut here, while he hasn't answered you. Maybe it can speed things a bit. So you get your PR merged sooner rather than later.
Hope it makes sense.

Reply:
i had natsukium solution in preCheck if that is not ok, i can still make the changes. Once again thank you for your feedback.

Reply:
Sorry for the lack of explanation.
If patched, the patch needs to be updated every time the source is updated.
Adding environment variables is a more common solution and a more stable way.